### ISSUE TITLE
Current build feels like playing in a desert

### ISSUE THREAD

**Issue by YehnBeep** [CleverRaven/Cataclysm-DDA#59590]
### Is your feature request related to a problem? Please describe.

Rain was much more frequent in older builds. Now it feels like finding a raincoat, funnel, towel and so on (or crafting your own) are just pointless.

### Solution you would like.

Increased frequency of inclement weather. (Possibly as a world setting?)

### Describe alternatives you have considered.

Um... file off the setting's serial number and say we're in Arizona instead?

**Comment by Bulletfang** [comment]
We have a setting for eternal season, but what about a setting for eternal rain?

**Comment by YehnBeep** [comment]
Well, it would be nice to have an option to have rain more frequently than once in a blue moon without also going so far as to make it permanent...

**Comment by Night-Pryanik** [comment]
There was an attempt at creating eternal weather: #52637.

**Comment by Night-Pryanik** [comment]
Weather in general and amount of rains in particular was overhauled a while ago in #34515. That PR was based on RL-data for rains in New England.

tl;dr In New England heavy rains (depicted as `Rain` in game) are very rare. Light rains are rather frequent, but since they are barely noticeable, they ain't simulated in the game at all. Thus we observe much smaller amount of rains overall.

> Well, it would be nice to have an option to have rain more frequently than once in a blue moon

You have this option. It's located in `regional_map_settings.json`, in the `weather` section. You can edit it to your liking, for example, convert New England into a heavy-rain-every-day tropical land, like in `TropiCataclysm` mod.

**Comment by YehnBeep** [comment]
That file looks very technical, and I don't understand how to edit those settings in a meaningful way. Would it be possible to get something like a slider in game that could make bad weather 2x more frequent, or 3x more frequent, and so on?

**Comment by Night-Pryanik** [comment]
Sure, it's possible. The thing is, project manager doesn't like adding options or sliders to world setting **if** they are already accessible through jsons.

**Comment by Zireael07** [comment]
@Night-Pryanik I think this is a similar case as the forest thickness PR - you can *technically* edit it in JSONs, but this is very involved. Just as with the forests, I think it warrants a slider on the new game setup screen.

Alternately light rain could be made to have at least some effect.

**Comment by Night-Pryanik** [comment]
As with forest thickness, I'm NOT saying I'm against it. I'm just saying that Kevin MIGHT dislike this.

**Comment by kevingranade** [comment]
Issue too vague. "Rain was more frequent" is not actionable. Neither is,  "finding x,y,z items seems pointless.
